.gitignore

```diff
@@ -1,3 +1,6 @@
+# webpack
+webpack.config.js
```

Comment:
**TODO FOR LOST POINTS**

You should not be gitignoring the webpack.config.js.